packages/components/navbar/src/utils.styles.ts

```diff
@@ -19,3 +22,17 @@ export const getGlowOnFocusStyles = (shadow: string = tokens.glowPrimary) => ({
     boxShadow: shadow,
   },
 });
+
+export const increaseHitArea = (minSize = '44px'): CSSObject => ({
+  position: 'relative',
+  '&:after': {
```

Comment:
I think this should be `::before` to render the element "below" the item.

Reply:
I was using `::before` before, but it conflicts with avatar that already has a `::before`, and functionally wise, both seems to work, or did you find an issue with the after??

Reply:
Hmm that does make sense. For the Avatar I had to use `::before` exactly because it would otherwise visually render the wrong way. I figured it would be the same here. I haven't actually tested the code yet 😄